src/assets/scss/_common.scss

```diff
@@ -1,6 +1,40 @@
+body {
+  font-family: verdana;
+  font-size: 16px;
+}
 .container {
   margin: 20px auto 20px auto;
   max-width: 500px;
+  .todo-form {
+    display: flex;
+    input {
```

Comment:
Keep input styles in separate files.
ie. ``assets/scsss/ui/forms/_input.scss``

Reply:
Done